File: sandbox.client/front/ts/model/ClientDetails.ts
import {Address} from "./Address";
import {Phone} from "./Phone";
import {Gender} from "./Gender";
import {Charm} from "./Charm";

export class ClientDetails {
  public id: number/*int*/ = -1;
  public name: string;
  public surname: string;
  public patronymic: string;
  public gender: Gender;
  public birthDate: number/*long*/;
  public charmId: number/*int*/;
  public charmsDictionary: Charm[];
  public factAddress: Address;
  public regAddress: Address;
  public phones: Phone[];
}

File: sandbox.controller/src/kz/greetgo/sandbox/controller/model/ClientDetails.java
package kz.greetgo.sandbox.controller.model;

import java.util.ArrayList;
import java.util.List;

public class ClientDetails {
  public int id;
  public String name;
  public String surname;
  public String patronymic;
  public String gender;
  public String birth_date;
  public int charm_id;
  public String charm;
  public String fAdressStreet;
  public String fAdressHouse;
  public String fAdressFlat;
  public String rAdressStreet;
  public String rAdressHouse;
  public String rAdressFlat;
  public List<String> homePhone = new ArrayList<>();
  public List<String> workPhone = new ArrayList<>();
  public List<String> mobilePhones = new ArrayList<>();
}
